core/src/main/java/com/scalar/db/api/TableMetadata.java

```diff
@@ -108,33 +111,42 @@ public LinkedHashSet<String> getClusteringKeyNames() {
   }
 
   /**
-   * Returns the specified clustering order
+   * Returns the clustering order of the specified clustering key column
```

Comment:
```suggestion
   * Returns the clustering order of the specified clustering-key column
```

Reply:
@josh-wong In the following data modeling document, we don't add the dash (`-`) between `clustering` (or `partition`) and `key`:
https://github.com/scalar-labs/scalardb/pull/1829

Should we still add the dash?

Reply:
Yes, please. In some of my suggestions in #1829, I added a dash between `clustering` and `key`. Since the phrase `clustering key` modifies a noun (`column`), we should have a dash (`clustering-key column`).

Reply:
Sure, thanks!